### PR TITLE
Translated the docs

### PR DESCRIPTION
**Pull request by attreyee-muk**
I have translated all the files in the intro folder. Please see and let me know if any changes need to be done.

### CODE REVIEW

**Comment by shubham1172** [comment]
@a-muk welcome to this repo, please fix the DCO check - https://github.com/brigadecore/docs-hindi/pull/2/checks?check_run_id=5417461151

**Comment by krancour** [comment]
@shubham1172 are you going to assist with Hindi translations?

@a-muk was the first to volunteer for that, so I made her the de facto code owner when I created this repository, but it would be ideal to have at least two owners because even an owner needs their own PRs reviewed by another owner.

lmk if you want that responsibility.

**Comment by shubham1172** [comment]
Sure @krancour, that works. I can review this PR in some time too. Thanks!

File: content/intro/index.md
Comment (shubham1172) [inline content/intro/index.md]:
```suggestion
सभी की ज़रूरतों को पूरा करने के लिए Brigade की मुक्य प्रलेखन को कुछ "टुकड़ों" में तोड़ा गया है:
```

File: content/intro/index.md
Comment (shubham1172) [inline content/intro/index.md]:
```suggestion
यह परिचयात्मक सामग्री जो दी गयी है, वह उन लोगो के लिए है जो Brigade में नैय है. इसमें कुछ भी गहराई में नहीं बताया गया है, बल्कि इस्कि एक उच्च स्तरीय अवलोकन की गई है की Brigade को कैसे तैनात करना है, उसके साथ कैसे इंटरैक्ट करना है, कैसे प्रोजेक्ट्स बनाना है, और घटनाओं को संभालने वाली स्क्रिप्ट कैसे लिखनी है|
```

File: content/intro/index.md
Comment (shubham1172) [inline content/intro/index.md]: "Brigade की कम्पलीट गाइड, कोर कॉम्पोनेन्ट, सिस्टम रोल्स एट्सेटरा आपको वाहा मिल जाएगी." please recheck this. It translates to "**You will get brigade's complete guide**, core components, system roles, etc. there", instead of "**There are complete guides to** Brigade's core components, scripting libraries, system roles and more."

Comment (shubham1172) [reply content/intro/index.md]: "इसमें ही शायद आपको आपका ज़्यादा समय डालना होगा" -> "इसमें ही शायद आप अपना ज़्यादा समय बिताना चाहते हैं।"

Comment (attreyee-muk) [reply content/intro/index.md]: "vaha mil jayegi" basically points out to विषय गाइड.

Comment (shubham1172) [reply content/intro/index.md]: "अगर आप अच्छे से यह पढ़े, तोह आपको काफी कुछ पता चल जायेगा Brigade के बारे में|" please recheck this too. It conveys "you will know a lot about Bridage" vs "you should come out knowing pretty much everything there is to know about Brigade". We can simplify the latter but meaning should not be lost.

Comment (attreyee-muk) [reply content/intro/index.md]: > "इसमें ही शायद आपको आपका ज़्यादा समय डालना होगा" -> "इसमें ही शायद आप अपना ज़्यादा समय बिताना चाहते हैं।"
Yeah, sure. That would sound better. Thank you!

Comment (shubham1172) [reply content/intro/index.md]: >"vaha mil jayegi" basically points out to विषय गाइड.

Yeah, that part is fine. I was talking about the first part. We are conveying that you will get the complete guide to brigade, core components, etc. as opposed to them getting a **complete guide to** the core components, other things, etc.

File: content/intro/overview.md
Comment (shubham1172) [inline content/intro/overview.md]:
```suggestion
Brigade एक इवेंट संचालित स्क्रिप्टिंग उपकरण है जो Kubernetes पर चलता है। इसका मतलब यह है:
```

File: content/intro/overview.md
Comment (shubham1172) [inline content/intro/overview.md]: Manage the host server, right? 
```suggestion
- Brigade के लिए आपको कोई होस्ट सर्वर को प्रबंधित करना ज़रुरी नहीं है।
```

File: content/intro/overview.md
Comment (shubham1172) [inline content/intro/overview.md]:
```suggestion
-Brigade विशेष रूप से सीआई और सीडी कार्यभार के लिए उपयुक्त है जैसे:
```

File: content/intro/overview.md
Comment (shubham1172) [inline content/intro/overview.md]: nit: one more indentation level

File: content/intro/overview.md
Comment (shubham1172) [inline content/intro/overview.md]: Have these 4 points been intentionally left in English? I would recommend translating them too.

File: content/intro/overview.md
Comment (shubham1172) [inline content/intro/overview.md:21]: Please recheck translation.

Comment (shubham1172) [inline content/intro/overview.md]:
```suggestion
- Brigade अपने एपीआई सर्वर के साथ आता है |
```

File: content/intro/overview.md
Comment (shubham1172) [inline content/intro/overview.md]: बातचीत != interaction. Also, let's use सीएलआई and एसडीके instead.

File: content/intro/overview.md
Comment (shubham1172) [inline content/intro/overview.md]: "अभी भी" looks out of place.

Comment (shubham1172) [inline content/intro/overview.md]: Inconsistency, let's use either brigade or ब्रिगेड throughout. I would suggest the latter.

File: content/intro/overview.md
Comment (shubham1172) [inline content/intro/overview.md]:
```suggestion
  - ब्रिगेड सर्वर को हेल्म के साथ तैनात और प्रबंधित किया जाता है, इस प्रकार होस्टिंग किसी भी स्टॉक कुबेरनेट्स सिस्टम पर हो सकती है, एज़्योर से लेकर मिनिक्यूब तक।
```

File: content/intro/overview.md
Comment (shubham1172) [inline content/intro/overview.md]: authn = प्रमाणीकरण
authz = प्राधिकरण

File: content/intro/overview.md
Comment (shubham1172) [inline content/intro/overview.md]: Same comment for English words.

File: content/intro/overview.md
Comment (shubham1172) [inline content/intro/overview.md]: "माध्यम से आसानी से बना रहता है" please recheck.

File: content/intro/overview.md
Comment (shubham1172) [inline content/intro/overview.md]: "यह कोई SaaS नहीं है".

"न ही इसे कॉन्फ़िगर करने और चलाने के लिए डोमेन विशेषज्ञों की एक सेना (or you can use समूह) की आवश्यकता है।" -> let's instead use

File: content/intro/readnext.md
Comment (shubham1172) [inline content/intro/readnext.md]:
```suggestion
जैसा कि आप प्रलेखन का अध्ययन करते हैं, यदि आपको कोई अंतर या क्षेत्र दीखायी पढ़े जिसे ध्यान देने की आवश्यकता हो, तो कृपया [समस्या दर्ज करके] हमें बताएं।
```

File: content/intro/quickstart.md
Comment (shubham1172) [inline content/intro/quickstart.md]:
```suggestion
## आवश्यकताएँ
```

**Review by shubham1172 — CHANGES_REQUESTED** [review]
@a-muk LGTM overall, kudos on the work with translation! I just had a few comments mostly 
 regarding consistency and uniformity (and a few translation misses). Please also fix the DCO check.

File: content/intro/overview.md
Comment (attreyee-muk) [reply content/intro/overview.md]: What should I write these in Hindi?

Comment (shubham1172) [reply content/intro/overview.md]: One option is to transliterate the proper nouns. 

For example,
1. "GitHub" becomes "गिटहब (GitHub)"
2. "Azure Active Directory" becomes "ऐज़ूर एक्टिव डायरेक्टरी (Azure Active Directory)"
3. "OAuth2" becomes "औ ौथ २ (OAuth2)"

In this case, the first occurrence always has an English version too.

The second option is to do the other way, i.e.,

For example,
1. "GitHub" becomes "GitHub (गिटहब)"
2. "Azure Active Directory" becomes "Azure Active Directory (ऐज़ूर एक्टिव डायरेक्टरी)"
3. "OAuth2" becomes "OAuth2 (औ ौथ २)"

This will be easier to maintain. Thoughts?

Comment (attreyee-muk) [reply content/intro/overview.md]: Okay, I have written many of such words (computer programming related) in English.

Comment (shubham1172) [reply content/intro/overview.md]: Yup, we will need to decide on a format. I believe the second option is better.

File: content/intro/index.md
Comment (attreyee-muk) [reply content/intro/index.md]: Can you please suggest me what should I write?

> > "vaha mil jayegi" basically points out to विषय गाइड.
> 
> Yeah, that part is fine. I was talking about the first part. We are conveying that you will get the complete guide to brigade, core components, etc. as opposed to them getting a **complete guide to** the core components, other things, etc.

File: content/intro/overview.md
Comment (attreyee-muk) [reply content/intro/overview.md]: Shall I just write the Hindi translations of it in brackets alongside?

File: content/intro/overview.md
Comment (attreyee-muk) [reply content/intro/overview.md]: Is it okay to write 'API' also in Hindi font?

File: content/intro/overview.md
Comment (attreyee-muk) [reply content/intro/overview.md]: Yeah, 2nd one seems better. I can start doing it , but it may take sometime because a lot has to be changed.

File: content/intro/overview.md
Comment (attreyee-muk) [reply content/intro/overview.md]: Okay

File: content/intro/overview.md
Comment (attreyee-muk) [reply content/intro/overview.md]: Ohh yes. I forgot to write Brigade instead of ब्रिगेड over here. Thank you for pointing it out!

File: content/intro/overview.md
Comment (attreyee-muk) [reply content/intro/overview.md]: What word can be used instead of बातचीत for interactions? Can we just write इन्टरैक्शन in that place?

File: content/intro/index.md
Comment (shubham1172) [reply content/intro/index.md]: Yeah sure. 
"Brigade की कम्पलीट गाइड, कोर कॉम्पोनेन्ट, सिस्टम रोल्स एट्सेटरा आपको वाहा मिल जाएगी." => "Brigade की कोर कॉम्पोनेन्ट, सिस्टम रोल्स, आदि, की पूर्ण गाइड आपको वाहा मिल जाएगी."

File: content/intro/overview.md
Comment (shubham1172) [reply content/intro/overview.md]: Yes, please take your time!

File: content/intro/overview.md
Comment (shubham1172) [reply content/intro/overview.md]: Yes, let's transliterate.

File: content/intro/overview.md
Comment (shubham1172) [reply content/intro/overview.md]: We can follow the same model that we decided. "API (एपीआई)"

File: content/intro/overview.md
Comment (shubham1172) [reply content/intro/overview.md]: "Automated testing" can be simply "स्वचालित परीक्षण".
If it were proper nouns or acronyms, we could go with the model that we decided on.

For some not-so-straightforward translations like "artifacts", we can also use the transliteration model.

File: content/intro/overview.md
Comment (attreyee-muk) [reply content/intro/overview.md]: Okay

File: content/intro/overview.md
Comment (attreyee-muk) [reply content/intro/overview.md]: So, will it not be like ignoring the the fact that it is not a SaaS?

Comment (shubham1172) [reply content/intro/overview.md]: Sorry, I didn't get your question?

Comment (attreyee-muk) [reply content/intro/overview.md]: Are you saying that I should totally delete the sentence यह कोई SaaS नहीं है ?

Comment (attreyee-muk) [reply content/intro/overview.md]: oh sorry my bad, thank you:)

Comment (shubham1172) [reply content/intro/overview.md]: No no, I meant "यह SaaS नहीं है" -> "यह **कोई** SaaS नहीं है"

Comment (attreyee-muk) [reply content/intro/overview.md]: Yeah, I got it. Thank you.

**Comment by attreyee-muk** [comment]
@shubham1172 just an update- I'm only left with correcting things in the quickstart.md file.

**Comment by shubham1172** [comment]
Thanks @a-muk, it's better to keep pushing your changes so that you don't accidentally lose them 😄 Also, the DCO needs to be fixed.

**Comment by attreyee-muk** [comment]
@shubham1172 Actually I just started with quickstart and made some mistakes which I have to undo. So, I will make a PR after I complete the whole thing. Only transliteration of some words is left.

**Comment by shubham1172** [comment]
Sure, sounds good! Thanks @a-muk.

**Comment by attreyee-muk** [comment]
@shubham1172 can you pls help me translate this line- "A Brigade [project](/topics/project-developers/projects) pairs event
subscriptions with worker (event handler) configuration" ?

**Comment by shubham1172** [comment]
> @shubham1172 can you pls help me translate this line- "A Brigade [project](/topics/project-developers/projects) pairs event subscriptions with worker (event handler) configuration" ?

Let's go with "एक ब्रिगेड प्रोजेक्ट वर्कर (ईवेंट हैंडलर) कॉन्फ़िगरेशन के साथ ईवेंट सब्स्क्रिप्शंस कॉन्फ़िगरेशन भी रखता है"

This is slightly changed from the original English line, but this seems like the intent of it.

**Comment by attreyee-muk** [comment]
@shubham1172  I signed off all the commits and also made a lot new changes. Please take your time and check the docs once.

File: CODEOWNERS
Comment (shubham1172) [inline CODEOWNERS]: Please revert this file changes.

**Comment by shubham1172** [comment]
> @shubham1172 I signed off all the commits and also made a lot new changes. Please take your time and check the docs once.

Thanks, I will review it, the DCO is still failing, can you please check the commits again?

**Comment by a-muk** [comment]
@shubham1172 Is it okay now?

**Comment by attreyee-muk** [comment]
I approved the DCO now.

File: CODEOWNERS
Comment (shubham1172) [inline CODEOWNERS]: Can you please reset this file?  https://stackoverflow.com/questions/37972753/git-reset-single-file-in-feature-branch-to-be-the-same-as-in-master

File: content/intro/index.md
Comment (shubham1172) [inline content/intro/index.md:3]: Nit, space before `(`
Edit: For all occurrences

Comment (shubham1172) [reply content/intro/index.md:3]: When we have `English (Hindi)` translation, we can do only once. From next time, we can stick to `English`.
For example, `Brigade (ब्रिगेड)` can be in the text once, and then we can only use `Brigade`.

**Review by shubham1172 — CHANGES_REQUESTED** [review]
Thank you for the PR @a-muk, this is really good! Let's just fix some nits and we can close this one.

**Comment by shubham1172** [comment]
@a-muk, did you get a chance to look at this?

File: content/intro/index.md
Comment (attreyee-muk) [reply content/intro/index.md:3]: So am not changing this now. Is that fine?

File: CODEOWNERS
Comment (attreyee-muk) [reply CODEOWNERS]: pathspec 'origin/master' did not match any file(s) known to git

I am getting this error. Can you please help me out?

Comment (attreyee-muk) [reply CODEOWNERS]: I can even make these changes and submit a pull request again or maybe a new pull request containing only those changes.

Comment (shubham1172) [reply CODEOWNERS]: Took it offline and fixed.

File: content/intro/index.md
Comment (shubham1172) [reply content/intro/index.md:3]: Fixed it.

File: CODEOWNERS
Comment (krancour) [inline CODEOWNERS:4]: fwiw, the "errors" here are my problem.

It's complaining that either that team does not exist or doesn't have write access to this repo and neither of those things are true. It may be a bug in GitHub itself and this same issue is affecting other repos as well.

**Comment by attreyee-muk** [comment]
@shubham1172 I think it's okay now. What do you think?

**Comment by shubham1172** [comment]
> @shubham1172 I think it's okay now. What do you think?

Can you please review and merge https://github.com/a-muk/docs-hindi/pull/1? Then we can close this.

**Comment by krancour** [comment]
@shubham1172 if you approve the PR, she'll be able to merge. It's blocked without approval.

**Comment by shubham1172** [comment]
> @shubham1172 if you approve the PR, she'll be able to merge. It's blocked without approval.

@krancour I am talking about merging my PR on @a-muk's PR first, it contains some cosmetic changes - https://github.com/a-muk/docs-hindi/pull/1. After that we can merge this PR too.

**Comment by krancour** [comment]
Ah. I misunderstood. Thanks for clarifying.

**Review by shubham1172 — APPROVED** [review]

**Comment by krancour** [comment]
@a-muk this would be mergeable now if the commits were verified. The easiest way to correct this is probably for you to squash all the commits and then amend the one remaining commit by following the instructions here: https://docs.brigade.sh/topics/contributor-guide/signing/

**Comment by shubham1172** [comment]
@krancour can you please help @a-muk here with the code-signing? We tried a few things offline but it did not seem to have worked.

**Comment by krancour** [comment]
@a-muk @shubham1172 instructions are here: https://youtu.be/uHkQzxzciLA?t=1590

That URL starts the video at the relevant time index.

File: content/intro/quickstart.md
Comment (anubha-v-ardhan) [inline content/intro/quickstart.md:172]:
```suggestion
एक Brigade(ब्रिगेड) [प्रोजेक्ट](/topics/project-developers/projects) वर्कर (ईवेंट हैंडलर) कॉन्फ़िगरेशन के साथ ईवेंट सब्स्क्रिप्शंस कॉन्फ़िगरेशन भी रखता है|
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

**Comment by shubham1172** [comment]
Since @a-muk seem to be unavailable, I can create a new PR with these changes and add @a-muk as a co-author. Will wait for another day or two.

**Comment by attreyee-muk** [comment]
Hey @shubham1172 it is okay if you create the PR and add me as a co-author since I'm unable to do the last procedures.

**Comment by shubham1172** [comment]
> Hey @shubham1172 it is okay if you create the PR and add me as a co-author since I'm unable to do the last procedures.

Yes, I will create a PR.

**Comment by shubham1172** [comment]
New PR created #6, closing this PR.

**Pull request closed without merging**
